ruby_on_rails/app_initialisation.md

```diff
@@ -137,18 +151,7 @@ create the initializer for figaro in `config/initializers/figaro.rb`:
 * Enable the default [Content Security Policies](https://github.com/renuo/applications-setup-guide/blob/master/ruby_on_rails/content_security_policy.md) in `config/initializers/content_security_policy.rb`.
   The report URI will be set later in the step of Sentry configuration.
 
-* If you're using Webpacker, let's clean up after asset precompilation
```

Comment:
Isn't this necessary somehow also when using esbuild, webpack or other js bundling systems?

Reply:
No because sprockets & propshaft provide this rake task by default. 

```ruby
rails assets:clean                       # Removes old files in config.assets.output_path
rails assets:clobber                     # Remove config.assets.output_path
```